HousingRepairsSchedulingApi/Services/Drs/DrsService.cs

```diff
@@ -68,7 +68,12 @@ public async Task<IEnumerable<AppointmentSlot>> CheckAvailability(string sorCode
             {
                 var checkAvailabilityResponse = await _drsSoapClient.checkAvailabilityAsync(new checkAvailability(checkAvailability));
 
-                _logger.LogInformation("Called checkAvailabilityAsync for {LocationId}, returning {Response}", locationId, JsonSerializer.Serialize(checkAvailabilityResponse));
+                _logger.LogInformation("Called checkAvailabilityAsync for {LocationId}, returning {Response}", locationId, checkAvailabilityResponse);
```

Comment:
This {Response}, should it be {checkAvailabilityResponse} ?

Reply:
Response is not a parameter, its a template string. Notice there is no dollar sign.